src/main/features/core/tray.js

```diff
@@ -40,9 +63,35 @@ const setContextMenu = () => {
 setContextMenu();
 
 appIcon.setToolTip('Google Play Music');
-appIcon.on('double-click', () => {
-  mainWindow.setSkipTaskbar(false);
-  mainWindow.show();
+
+
+// Iconified     -> show, to foreground
+// In background -> to foreground
+// In foreground -> iconify
+appIcon.on('click', () => {
+  if (process.platform === 'darwin') {
+    // Special code for OS-X - Not tested!
+    if (!mainWindow.isVisible()) {
+      mainWindow.setSkipTaskbar(false);
+      mainWindow.show();
+    } else {
+      if (PlaybackAPI.isPlaying()) {
+        Emitter.sendToGooglePlayMusic('playback:playPause');
```

Comment:
This seems wrong but I can't test until I get home.  We shouldn't be pausing the player by clicking the tray icon